code/game/turfs/turf.dm

```diff
@@ -261,6 +261,7 @@
 /turf/indestructible
 	name = "wall"
 	icon = 'icons/turf/walls.dmi'
+	density = 0

```

Comment:
![you tried](https://cloud.githubusercontent.com/assets/5573038/7973492/7fc34e76-0a1b-11e5-902c-c5c167555888.png)